pom.xml

```diff
@@ -372,7 +372,14 @@
       </build>
     </profile>
   </profiles>
-
+  
+  <repositories>
+    <repository>
+      <id>terracotta-releases</id>
+      <url>http://www.terracotta.org/download/reflector/releases</url>
+    </repository>
+  </repositories>
```

Comment:
This is to pick up terracotta-apis.  where it is deployed is elsewhere

Reply:
but should we then deploy api in central to not have a mix of dependencies in central and not in central ? 
Because other people are using or might use angela and they point to maven central.

Either we remove angela from central (and force users to add the tc repo) or we move deps in central too I think.

Up to @aurbroszniowski @akomakom @chrisdennis ...

Reply:
i agree its odd but I think its a mistake born out of history.  this problem already exists between core and platform so 🤷

Reply:
@aurbroszniowski told us some people are using Angela too, externally, and I think there are also some internal users at SAG.

Reply:
there will not be build problems if that is your concern since this repository reference is here, the tc-apis will just be pulled during build from the correct place.

Reply:
I am concerned about people (external and internal) who are consuming angela from here: https://repo1.maven.org/maven2/org/terracotta/angela/

Reply:
why will it fail?  It builds on azure.  Maven resolves the dependencies in the repositories provided.  Any user of maven central will resolve the same as azure.

Reply:
yes it will fail because the new dependency won't be resolved by maven normally.

Let's take this sue case:

- angela is in maven central.
- I am working at XYZ and I have in my projet's pom:
```
      <dependency>
        <groupId>org.terracotta</groupId>
        <artifactId>angela</artifactId>
        <version>${angela.version}</version>
      </dependency>
```
Everything is fine.

But now I am incrementing to the new release, which now has a dependency to `standard-cluster-services`.

This dependency is not in central (https://repo1.maven.org/maven2/org/terracotta/). So Maven won't be able to resolve it and his build will fail.

The `repository` section of the pom is there to tell where to find the libs to build the project, not to tell downstream projects where to resolve the dependencies.

It does not change anything for us at Tc because we have a settings.xml that is pointing  to our nexus / tc repo. So we will resolve. But adding a custom repo outside maven central to find a dep that is not in central will force external users to also do the same and add the tc repo in their pom or nexus or settings file. 

This is a bad practise IMO.

There are some rules / guidelines regarding uploading deps in central: https://maven.apache.org/repository/guide-central-repository-upload.html

> I have other repositories or pluginRepositories listed in my POM, is that a problem?
At present, this won't preclude your project from being included, but we do strongly encourage making sure all your dependencies are included in the Central Repository. If you rely on sketchy repositories that have junk in them or disappear, it just creates havok for downstream users. Try to keep your dependencies among reliable repos like Central, Jboss, etc.

Reply:
I can change to using reflexive calls instead of the new dependency for now.  Long term Angela may need to be restructured to allow the DistributionControllers to be defined closer to the kits they control.  Updating things like command-line options would be easier as well.

Reply:
the other option is to make the dependency in `provided` scope (or much more `optional`). optional is in the case of an optionally activated feature, which is the case here.

Reply:
yeah that is better and we can discuss later a more permanent solution.